www/docs/nextjs/introduction.md

```diff
@@ -212,17 +212,11 @@ export default function IndexPage() {
 
 The `config`-argument is a function that returns an object that configures the tRPC and React Query clients. This function has a `ctx` input that gives you access to the Next.js `req` object, among other things. The returned value can contain the following properties:
 
-- Exactly **one of** these are **required**:
-
-  - `url` your API URL.
+- **Required**:
   - `links` to customize the flow of data between tRPC Client and the tRPC-server. [Read more](../client/links.md).
-
 - Optional:
   - `queryClientConfig`: a configuration object for the React Query `QueryClient` used internally by the tRPC React hooks: [QueryClient docs](https://tanstack.com/query/v4/docs/reference/QueryClient)
-  - `headers`: an object or a function that returns an object of outgoing tRPC requests
   - `transformer`: a transformer applied to outgoing payloads. Read more about [Data Transformers](data-transformers)
-  - `fetch`: customize the implementation of `fetch` used by tRPC internally
```

Comment:
`headers`, `fetch`, and `AbortController` are managed within `links` now. Is there an update to the documentation you would like to see for that?

Reply:
We should do an overhaul of https://trpc.io/docs/v10/links